pkg/injector/webhook.go

```diff
@@ -57,14 +57,14 @@ func NewWebhook(config Config, kubeClient kubernetes.Interface, certManager cert
 	}
 
 	wh := webhook{
-		config:              config,
-		kubeClient:          kubeClient,
-		certManager:         certManager,
-		meshCatalog:         meshCatalog,
-		namespaceController: namespaceController,
-		osmNamespace:        osmNamespace,
-		cert:                cert,
-		configurator:        cfg,
+		config:       config,
+		kubeClient:   kubeClient,
+		certManager:  certManager,
+		meshCatalog:  meshCatalog,
+		Controller:   Controller,
```

Comment:
No need to export this.

```suggestion
		kubeController:   Controller,
```